cmd/queue/main.go

```diff
@@ -272,22 +265,23 @@ func buildServer(ctx context.Context, env config, drainer *pkghandler.Drainer, p
 	if tracingEnabled {
 		composedHandler = tracing.HTTPSpanMiddleware(composedHandler)
 	}
-	if env.ServingEnableRequestLog {
-		composedHandler = requestLogHandler(logger, composedHandler, env)
+
+	drainer := &pkghandler.Drainer{
+		QuietPeriod: drainSleepDuration,
+		// Add Activator probe header to the drainer so it can handle probes directly from activator
+		HealthCheckUAPrefixes: []string{network.ActivatorUserAgent},
+		Inner:                 composedHandler,
+		HealthCheck:           health.ProbeHandler(probeContainer, tracingEnabled),
 	}
+	composedHandler = drainer
```

Comment:
This line here confused me a little, why are we setting the drainer to the composedHandler instead of adding the requestLogHandler to the drainer and return that

Reply:
just to be consistent with the entire rest of the chain that wraps composedHandler each time rather than renaming it

Reply:
(and because otherwise we'd have to do `drainer = requestLogHandler(logger, drainer, env)` two lines down, which would be weird because it would no longer be a drainer)

Reply:
cool, trying to make sure I understand correctly :) thx